ui/src/app/views/workspace-nav-bar/component.css

```diff
@@ -58,6 +60,12 @@
   position: absolute !important;
   right: 0;
   height: 100%;
+  width: 5rem
+}
+
+::ng-deep #workspace-top-nav-bar .dropdown-menu {
```

Comment:
Nope. Because we don't have control over the clarity stuff otherwise... That's why I tagged with the ID and am targeting children of objects with the ID only.